extensions/amp-analytics/amp-analytics.md

```diff
@@ -164,6 +164,26 @@ Type attribute value: `mediametrie`
 
 Adds support for Médiamétrie tracking pages. Requires defining *var* `serial`. Vars `level1` to `level4` are optional.
 
+### OEWA
+
+Type attribute value: `oewa`
```

Comment:
Maybe move the last paragraph here? If I were a publisher, I probably would not get to that para to see that there is a faster, (arguably) better alternative available.